packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx

```diff
@@ -198,6 +201,7 @@ export class DetailsRow extends BaseComponent<IDetailsRowProps, IDetailsRowState
         componentRef={ this._resolveRef('_focusZone') }
         role='row'
         aria-label={ ariaLabel }
+        ariaDescribedBy={ ariaDescribedBy }
```

Comment:
`aria-describedby` for consistency?

Reply:
@cschleiden The focusZone for which we are adding the props here has ```ariaDescribedBy``` for aria-describedby. Hence, directly adding here instead, leveraged the property already part of FocusZone component

Reply:
okay.